src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -64,8 +64,19 @@
 				>
 					I want to receive updates about my loans, Kiva news, and promotions in my inbox
 				</kv-base-input>
+				<re-captcha-enterprise
+					:requried="needsCaptcha"
+					@update="captcha = $event"
```

Comment:
Wondering if we can tap into this update event to extract errors or rate limit failures for logging?

Reply:
Yeah, we might be able to. I'll look at that after verifying